### PR TITLE
Added links to NuGet package services

### PR DESCRIPTION
**Pull request by pkulikov**

### CODE REVIEW

File: docs/standard/library-guidance/nuget.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ ms.date: 10/02/2018
 ---
 # NuGet
 
-NuGet is a package manager for the .NET eco-system and is the primary way developers discover and acquire .NET open-source libraries. NuGet.org, a free service provided by Microsoft for hosting NuGet packages, is the primary host for public NuGet packages but you can publish to custom NuGet services like MyGet and Azure DevOps.
+NuGet is a package manager for the .NET eco-system and is the primary way developers discover and acquire .NET open-source libraries. [NuGet.org](https://www.nuget.org/), a free service provided by Microsoft for hosting NuGet packages, is the primary host for public NuGet packages but you can publish to custom NuGet services like [MyGet](https://www.myget.org/) and [Azure DevOps](https://azure.microsoft.com/services/devops/artifacts/).
```
Comment (pkulikov) [inline docs/standard/library-guidance/nuget.md]: Given the service has a name, should the link "Azure DevOps" be renamed to "Azure Artifacts"?

**Review by rpetrusha — CHANGES_REQUESTED** [review]
In addition to the change suggested by @pkulikov, there are also two other issues, though both are unrelated to your PR. 
- eco-system --> ecosystem
- comma before "but you can publish" 
I'd appreciate it if you could make those changes as well.

**Comment by pkulikov** [comment]
@rpetrusha thanks for review; I've also addressed additional feedback.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thank you so much, @pkulikov, for making the additional, extraneous changes. I really appreciate it. I'll merge your PR now.

**Pull request merged**